cpp/inc/bond/core/schema.h

```diff
@@ -261,16 +265,26 @@ class InitSchemaDef
     template <typename T>
     uint16_t GetStructDef() const
     {
-        size_t n;
+        const auto& structs = _schema.structs;
+
+        BOOST_ASSERT(structs.size() <= (std::numeric_limits<uint16_t>::max)());
+
+        auto it = std::find_if(
+            std::cbegin(structs),
```

Comment:
Apparently `cbegin` is [missing](https://travis-ci.org/Microsoft/bond/jobs/327429088) from the standard library we're using in Travis.